barretenberg/cpp/src/barretenberg/translator_vm/translator_relations_definitions.hpp

```diff
@@ -0,0 +1,5 @@
+#include "barretenberg/relations/translator_vm/translator_decomposition_relation_impl.hpp"
+#include "barretenberg/relations/translator_vm/translator_delta_range_constraint_relation_impl.hpp"
```

Comment:
i dont feel this helps that much as you still need to explicitly list the relations whenever you would use this
also, headers should always start with #pragma once as a matter of style if nothing else